docs/getting-started/configuration.md

```diff
@@ -6,18 +6,18 @@ Seldon can be configured via various config files.
 
 We allow configuration of the Kafka integration. In general this configuration looks like:
 
-```{literalinclude} ../../../../../scheduler/config/kafka-internal.json
+```{literalinclude}
```

Comment:
is this now missing the content of the include?

Reply:
All hard-coded references have been fixed. See output here https://docs.seldon.ai/seldon-core-v2/getting-started/configuration